apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/[keyId]/page.tsx

```
@@ -25,6 +25,9 @@ import ms from "ms";
 import { notFound } from "next/navigation";
 import { Chart } from "./chart";
 import { VerificationTable } from "./verification-table";
+import RolePermissionsTree from "./permission-list";
+import { NestedPermissions } from "@/app/(app)/authorization/roles/[roleId]/tree";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)